lib/Runtime/Library/RegexHelper.cpp

```diff
@@ -347,9 +347,109 @@ namespace Js
         UnifiedRegex::Matcher* matcher;
     };
 
+    template <bool updateHistory>
+    Var RegexHelper::RegexMatchImpl(ScriptContext* scriptContext, RecyclableObject *thisObj, JavascriptString *input, bool noResult, void *const stackAllocationPointer)
+    {
+        ScriptConfiguration const * scriptConfig = scriptContext->GetConfig();
```

Comment:
nit: why not create local for just IsES6RegExSymbolsEnabled()? I see scriptConfig is used for only that purpose.